src/AdminConsole/package.json

```
@@ -11,10 +11,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "tailwindcss": "^3.4.0",
     "apexcharts": "3.45.1",
     "es-module-shims": "1.8.2",
-    "tailwindcss": "3.4.0",
+    "tailwindcss": "^3.4.0",
     "vue": "3.4.10"
+  },
+  "devDependencies": {
+    "@tailwindcss/aspect-ratio": "^0.4.2",
+    "@tailwindcss/forms": "^0.5.7",
+    "@tailwindcss/typography": "^0.5.10"
```

Renovate suggests pinning dependencies:

```suggestion
    "tailwindcss": "3.4.0",
    "vue": "3.4.10"
  },
  "devDependencies": {
    "@tailwindcss/aspect-ratio": "0.4.2",
    "@tailwindcss/forms": "0.5.7",
    "@tailwindcss/typography": "0.5.10"
```